static/app/gettingStartedDocs/java/spring.tsx

```diff
@@ -129,6 +147,10 @@ const getMavenInstallSnippet = (params: Params) => `
   ...
 </build>`;
 
+const getOpenTelemetryRunSnippet = (params: Params) => `
+SENTRY_AUTO_INIT=false java -javaagent:sentry-opentelemetry-agent-${getPackageVersion(params, 'sentry.java.opentelemetry-agent', '8.0.0')}.jar -jar your-application.jar
```

Comment:
Should we add some sort of info here, that the agent needs to be attached to the application server?